### DISCUSSION TITLE
Can't get it to work on pixel 6

### DISCUSSION THREAD

**Discussion by Homeslicer** [skyline-emu/skyline#1180] [General]
Anyone else on the same boat?

**Comment by Crytonics** [comment]
You have mali gpu, you need to wait until they fix for mali gpu. 
Thats why your games dosent work or have broken rendering.
They working on fixes but Adreno gpu is better for emulation than mali gpu.

**Reply by Homeslicer** [reply]
Thank you! l should've stuck with Samsung😥

**Reply by Crytonics** [reply]
Well if you live in europe then is the same because europe versions of samsung have Exynos cpu and that cpu dosent have Adreno gpu.

**Comment by Homeslicer** [comment]
I'm in the US i think they use snapdragon and adreno

**Reply by Crytonics** [reply]
Yea for US is Snapdragon.

**Reply by Lingomancer** [reply]
Yeah, Pixel 6 using Mali G78 MP20, which is actually a beast of a GPU but supposedly poor driver implementation makes emulation on it kind of rough.
